### PR TITLE
Filter products

### PR DESCRIPTION
**Pull request by Chris-Hesterman**
Filter out main product id from traits cards.  Correct some minor css artifacts that seem to come back.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -12,58 +13,66 @@ app.use(bodyParser.urlencoded({ extended: true }));
...
       .then((resultsFinal) => {
+        // console.log('-----rFinal', resultsFinal);
         resultsFinal.forEach((result) => {
+          if (result.products.indexOf(id) >= 0) {
+            const filteredProducts = result.products.filter((product) => {
+              return product !== id;
+            });
+            // eslint-disable-next-line no-param-reassign
+            result.products = filteredProducts;
+          }
```
Comment (RaneWallin) [inline server/index.js:33]: I think lines 27 - 33 can be simplified to `result.products = result.products.filter(product => product !== id)`

File: server/index.js [hunk 2]
```diff
@@ -12,58 +13,66 @@ app.use(bodyParser.urlencoded({ extended: true }));
...
             return productArray;
           })
           .then((resToClient) => {
             res.set({ 'Access-Control-Allow-Origin': '*' });
-            res.send(resToClient);
+            res.status(200).send(resToClient);
           })
           .catch((err) => {
-            throw err;
+            res.status(505).send('there was a problem');
```
Comment (RaneWallin) [inline server/index.js:75]: There's no status 505

Comment (Chris-Hesterman) [reply server/index.js:75]: well I can't argue with that

File: server/index.js (hunk 1 above)
Comment (Chris-Hesterman) [reply server/index.js:33]: yes, although the lint rules complain. I'll change it

File: server/index.js (hunk 2 above)
Comment (Chris-Hesterman) [reply server/index.js:75]: yes, thank you ! fixed.  500

File: server/index.js (hunk 1 above)
Comment (Chris-Hesterman) [reply server/index.js:33]: Agreed, thank you! changed

**Comment by Chris-Hesterman** [comment]
Still need to fix testing and do more, I haven't forgotten.

**Pull request merged**